lms/djangoapps/grades/signals/handlers.py

```diff
@@ -300,3 +305,20 @@ def listen_for_course_grade_passed_first_time(sender, user_id, course_id, **kwar
     """
     events.course_grade_passed_first_time(user_id, course_id)
     events.fire_segment_event_on_course_grade_passed_first_time(user_id, course_id)
+
+
+@receiver(EXAM_ATTEMPT_VERIFIED)
+def exam_attempt_verified_event_handler(sender, signal, **kwargs):  # pylint: disable=unused-argument
+    """
+    Consume `EXAM_ATTEMPT_VERIFIED` events from the event bus. This will trigger
+    an undo section override, if one exists.
+    """
+    from ..api import should_override_grade_on_rejected_exam, undo_override_subsection_grade
```

Comment:
And this is also related to the convo yesterday about needing to do the import inline?

Reply:
Yes! This is what we chatted about in the huddle - because the python API for this service is meant to be used both externally and internally, we end up with some circular dependencies. To avoid that, we have to use an inline import here